docs/make.jl

```diff
@@ -69,6 +69,7 @@ end
 
 makedocs(;
          modules = [DispersiveShallowWater, TrixiBase],
+         checkdocs = :exports,
          authors = "Joshua Lampert <joshua.lampert@uni-hamburg.de>",
```

Comment:
```suggestion
```
I think that's not needed and I don't even see any documentation about a `authors` keyword in the docs of Documenter.jl.